docs/modules/ROOT/pages/explanations/exoscale/limitations.adoc

```diff
@@ -0,0 +1,62 @@
+= Limitations on Exoscale
+
+[abstract]
+.Abstract
+Doing a UPI on Exoscale as xref:how-tos/install/exoscale.adoc[documented here] comes with some limitations which are outlined in this document.
+
+
+== Internal API Load Balancer
+
+One of the https://docs.openshift.com/container-platform/4.5/installing/installing_bare_metal/installing-bare-metal.html#installation-network-user-infra_installing-bare-metal[network topology requirements] for a UPI is a load balancer which is accessible internally of the cluster.
+This Load balancer is used to access both the Kubernetes API and the ignition endpoint.
+
+There are several managed offerings that could be consideren to solve this on Exoscale.
+
+=== Managed Elastic IPs
+
+Using a https://community.exoscale.com/documentation/compute/eip/#managed-eip[managed EIP] would allow to configure health checks and therefore automate the failover of one master to the other.
+
+[quote, 'https://community.exoscale.com/documentation/compute/eip/#limitations[Exoscale documentation]']
+____
+Instances sharing a common EIP can't communicate with each other by using the EIP address
+____
+
+Therefore we can't use them because the master nodes wouldn't be able to communicate via it.
+
+=== Network Load Balancer
+
+Using a https://community.exoscale.com/documentation/compute/network-load-balancer[Network Load Balancer] only allows to https://community.exoscale.com/documentation/compute/network-load-balancer/#limitations_1[target instance pools].
+Since during installation, the bootstrap node needs to be included in the load balancing, we can't use an instance pool.
+
+=== Current Workaround
+
+We use DNS round robin to include the public IP of each master node and during installation also the bootstrap node in a DNS name.
```

Comment:
```suggestion
We use DNS round robin to include the IP of each master node and during installation also the bootstrap node in a DNS name.
```

This would implie that there is a private IP.